romancal/resample/resample.py

```diff
@@ -401,20 +412,82 @@ def resample_variance_array(self, name, output_model):
 
         setattr(output_model, name, output_variance)
 
-    def update_exposure_times(self, output_model):
+    def resample_exposure_time(self, output_model):
+        """Resample the exposure time from ``self.input_models`` to the ``output_model``.
+
+        Create an exposure time image that is the drizzled sum of the input
+        images.
+        """
+        output_wcs = output_model.meta.wcs
+        exptime_tot = np.zeros(output_model.data.shape, dtype="f4")
+
+        log.info("Resampling exposure time")
+        for model in self.input_models:
+            exptime = np.full(
+                model.data.shape, model.meta.exposure.effective_exposure_time
+            )
+
+            # create a unit weight map for all the input pixels with science data
+            inwht = resample_utils.build_driz_weight(
+                model, weight_type=None, good_bits=self.good_bits
+            )
+
+            resampled_exptime = np.zeros_like(output_model.data)
+            outwht = np.zeros_like(output_model.data)
+            outcon = np.zeros_like(output_model.context, dtype="i4")
+            # drizzle wants an i4, but datamodels wants a u4.
+
+            xmin, xmax, ymin, ymax = resample_utils.resample_range(
+                exptime.shape, model.meta.wcs.bounding_box
+            )
+
+            # resample the exptime array
+            self.drizzle_arrays(
+                exptime * u.s,  # drizzle_arrays expects these to have units
+                inwht,
+                model.meta.wcs,
+                output_wcs,
+                resampled_exptime,
+                outwht,
+                outcon,
+                pixfrac=1,  # for exposure time images, always use pixfrac = 1
+                kernel=self.kernel,
+                fillval=0,
+                xmin=xmin,
+                xmax=xmax,
+                ymin=ymin,
+                ymax=ymax,
+            )
+
+            exptime_tot += resampled_exptime.value
+
+        return exptime_tot
+
+    def update_exposure_times(self, output_model, exptime_tot):
         """Update exposure time metadata (in-place)."""
-        total_exposure_time = 0.0
+        m = exptime_tot > 0
+        if np.any(m):
+            total_exposure_time = np.median(exptime_tot[m])
```

Comment:
I'm not sure we should label the median exposure time as the total exposure time. 
Would is make sense to add an attribute median_exposure_time?

Reply:
Yeah, changing this is part of the L3 metadata updates; I was just filling up columns.  OTOH, the exposure time being computed is the total exposure time for the median pixel with at least some exposure time, so it's not totally crazy---but I agree that we should update the metadata.  Do you think that should be part of this PR?

Reply:
I don't think this is urgent. 

I'm not to worried about that we can define it this way. I'm more concerned that some automated process will look for total_exposure_time and generate dubious results. 

We should probably have a larger discussion with others, INS, SE?, archive?, before making a decision and finalizing the attributes.

Reply:
I think this would need a larger discussion with the other components, INS, SE?, Archive? 
So I'd go ahead and merge this and write a follow up ticket.